scout/server/blueprints/variants/templates/variants/str-variants.html

```diff
@@ -131,22 +131,26 @@
     <div class="form-group text-center">
       {% if more_variants %}
         <div class="row">
-          <div class="col-6">
-            <a class="btn btn-outline-secondary mx-auto d-block" href="{{ url_for('variants.str_variants', institute_id=institute._id, case_name=case.display_name, page=(page + 1), **form.data) }}">
-              Next page
-            </a>
-          </div>
-          <div class="col-6">
-            <a class="btn btn-outline-secondary mx-auto d-block" href="{{ url_for('variants.str_variants', institute_id=institute._id, case_name=case.display_name, page=1, **form.data) }}">
-              First page
-            </a>
+            <div class="col-6">
+                <button class="btn btn-outline-secondary mx-auto d-block"
+                  href="{{ url_for('variants.str_variants', institute_id=institute._id, case_name=case.display_name, page=1, **form.data) }}">
+                  First page
+                </button>
+              </div>
+              <div class="col-6">
+                <button class="btn btn-outline-secondary mx-auto d-block"
+                  href="{{ url_for('variants.str_variants', institute_id=institute._id, case_name=case.display_name, page=(page + 1), **form.data) }}">
+                  Next page
+                </button>
+              </div>
           </div>
         </div>
       {% else %}
         <i class="text-muted">No more variants to display</i>
-        <a class="btn btn-outline-secondary mx-auto d-block" href="{{ url_for('variants.str_variants', institute_id=institute._id, case_name=case.display_name, page=1, **form.data) }}">
-          First page
-        </a>
+        <button class="btn btn-outline-secondary mx-auto d-block"
```

Comment:
Button with href! Is that's new? :blush: I thought you had to either use an anchor, or use like an on_click function.

Reply:
Yeah, that doesn't work on stage. Either keep the link, adjust the form href or script the button. I would have done the first of the three, but anything that works! 😸

Reply:
👍